docker/airflow/1.10.5/apks/py3-numpy/numpy-1.17.0-musl.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/numpy/core/include/numpy/npy_endian.h b/numpy/core/include/numpy/npy_endian.h
```

Comment:
@ashb what is this file for?

Reply:
I honestly don't know - it was the patch that Alpine apply when they build their py3-numpy@edge package that we were installing before.